cli/tests/testdata/test/aggregate_error.out

```diff
@@ -1,7 +1,7 @@
 running 1 test from ./test/aggregate_error.ts
 aggregate ... FAILED ([WILDCARD])
 
- ERRORS 
+ ERRORS
```

Comment:
Also, removed this trailing whitespace. I have my editor setup to automatically remove trailing whitespace and this existing didn't seem useful.